src/generic/course-card/index.tsx

```diff
@@ -30,18 +27,19 @@ export const CourseCard = ({ course }: CourseCardProps) => {
       <Card.ImageCap
         src={getFullImageUrl(course.data.imageUrl)}
         fallbackSrc={noCourseImg}
-        srcAlt={course.data.content.displayName}
-        logoSrc={course.data.orgImg ? getFullImageUrl(course.data.orgImg) : undefined}
-        fallbackLogoSrc={!course.data.orgImg && noOrgImg}
+        srcAlt={`${course.data.content.displayName} ${course.data.number}`}
+        logoSrc={course.data.orgImageUrl ? getFullImageUrl(course.data.orgImageUrl) : undefined}
+        fallbackLogoSrc={!course.data.orgImageUrl && noOrgImg}
         logoAlt={course.data.org}
       />
       <Card.Section>
```

Comment:
Still waiting to hear back about the design decision around course number, but I was able to get this to match the designs decently well without custom styles by using
```jsx
      <Card.Header
        title={course.data.content.displayName}
        subtitle={course.data.org}
        size="sm"
      />
      <Card.Section />
      <Card.Footer textElement={startDateDisplay && intl.formatMessage(messages.startDate, {
        startDate: startDateDisplay,
      })}
      />
```

<img width="726" height="652" alt="image" src="https://github.com/user-attachments/assets/78da15b7-48e2-4a9f-a520-a2ff04a426f6" />

Reply:
I tried another thing and got to what feels somewhat reasonable design-wise.

```jsx
      <Card.Header
        title={course.data.content.displayName}
        subtitle={(
          <>
            <div>{course.data.number}</div>
            <div>{course.data.org}</div>
          </>
        )}
        size="sm"
      />
      <Card.Section />
      <Card.Footer
        textElement={
          startDateDisplay
          && intl.formatMessage(messages.startDate, {
            startDate: startDateDisplay,
          })
        }
      />
``` 

<img width="1480" height="744" alt="image" src="https://github.com/user-attachments/assets/ad4df4e7-cafc-40fc-a439-d38948d4da94" />

Reply:
I left [a comment on the issue](https://github.com/openedx/frontend-app-catalog/issues/10#issuecomment-3292500734) since the badge component used in the Figma design doesn't match the Paragon component.

Once were have design sign off for that I'm happy to close that issue.